wire-ios-sync-engine/Tests/Source/UserSession/ZMClientRegistrationStatusTests.swift

```diff
@@ -113,6 +113,41 @@ extension ZMClientRegistrationStatusTests {
         XCTAssertEqual(self.sut.currentPhase, .generatingPrekeys)
     }
 
+    func testThatItReturnsRegisteringMLSClient_IfIfMLSIsEnabledAfterRegisteringProteusClient() {
+        // given
+        let selfUser = ZMUser.selfUser(in: syncMOC)
```

Comment:
syncMoc.perform?

Reply:
Ok, turns out all tests were broken in this way. I cleaned up all of them.